--- init/initdb.sql.orig	2020-03-29 17:47:10.732328565 -0700
+++ init/initdb.sql	2020-03-29 19:20:02.227890869 -0700
@@ -754,26 +754,27 @@
 --
 
 -- Create default user "guacadmin" with password "guacadmin"
-INSERT INTO guacamole_entity (name, type) VALUES ('guacadmin', 'USER');
-INSERT INTO guacamole_user (entity_id, password_hash, password_salt, password_date)
+INSERT INTO guacamole_entity (name, type) VALUES ('guacadmin@guacadmin', 'USER');
+INSERT INTO guacamole_user (entity_id, email_address, password_hash, password_salt, password_date)
 SELECT
     entity_id,
+    'guacadmin@guacadmin',
     decode('CA458A7D494E3BE824F5E1E175A1556C0F8EEF2C2D7DF3633BEC4A29C4411960', 'hex'),  -- 'guacadmin'
     decode('FE24ADC5E11E2B25288D1704ABE67A79E342ECC26064CE69C5B3177795A82264', 'hex'),
     CURRENT_TIMESTAMP
-FROM guacamole_entity WHERE name = 'guacadmin' AND guacamole_entity.type = 'USER';
+FROM guacamole_entity WHERE name = 'guacadmin@guacadmin' AND guacamole_entity.type = 'USER';
 
 -- Grant this user all system permissions
 INSERT INTO guacamole_system_permission (entity_id, permission)
 SELECT entity_id, permission::guacamole_system_permission_type
 FROM (
     VALUES
-        ('guacadmin', 'CREATE_CONNECTION'),
-        ('guacadmin', 'CREATE_CONNECTION_GROUP'),
-        ('guacadmin', 'CREATE_SHARING_PROFILE'),
-        ('guacadmin', 'CREATE_USER'),
-        ('guacadmin', 'CREATE_USER_GROUP'),
-        ('guacadmin', 'ADMINISTER')
+        ('guacadmin@guacadmin', 'CREATE_CONNECTION'),
+        ('guacadmin@guacadmin', 'CREATE_CONNECTION_GROUP'),
+        ('guacadmin@guacadmin', 'CREATE_SHARING_PROFILE'),
+        ('guacadmin@guacadmin', 'CREATE_USER'),
+        ('guacadmin@guacadmin', 'CREATE_USER_GROUP'),
+        ('guacadmin@guacadmin', 'ADMINISTER')
 ) permissions (username, permission)
 JOIN guacamole_entity ON permissions.username = guacamole_entity.name AND guacamole_entity.type = 'USER';
 
@@ -782,9 +783,9 @@
 SELECT guacamole_entity.entity_id, guacamole_user.user_id, permission::guacamole_object_permission_type
 FROM (
     VALUES
-        ('guacadmin', 'guacadmin', 'READ'),
-        ('guacadmin', 'guacadmin', 'UPDATE'),
-        ('guacadmin', 'guacadmin', 'ADMINISTER')
+        ('guacadmin@guacadmin', 'guacadmin@guacadmin', 'READ'),
+        ('guacadmin@guacadmin', 'guacadmin@guacadmin', 'UPDATE'),
+        ('guacadmin@guacadmin', 'guacadmin@guacadmin', 'ADMINISTER')
 ) permissions (username, affected_username, permission)
 JOIN guacamole_entity          ON permissions.username = guacamole_entity.name AND guacamole_entity.type = 'USER'
 JOIN guacamole_entity affected ON permissions.affected_username = affected.name AND guacamole_entity.type = 'USER'
